src/appleseed.cli/stdouttilecallback.cpp

```diff
@@ -124,14 +129,37 @@ namespace
 #ifdef _WIN32
             const int old_stdout_mode = _setmode(_fileno(stdout), _O_BINARY);
 #endif
+            send_header(*frame, tile_x, tile_y);
+            send_tile(*frame, tile_x, tile_y);
+#ifdef _WIN32
+            _setmode(_fileno(stdout), old_stdout_mode);
+#endif
+        }
 
+      private:
+        // Do not change the values of the enumerators as this WILL break client compabitility.
+        enum ChunkType
+        {
+            ChunkTypeTileData       = 1,
+            ChunkTypeTileHighlight  = 2
+        };
+
+        boost::mutex m_mutex;
+
+        const bool m_single_plane;
```

Comment:
It's not clear to me what this means and what's its purpose, but I suppose it'll become clear later?

Reply:
Yes, it's like in the houdini callback. When `true`, aovs would be sent in stdout